packages/rewriter/package.json

```diff
@@ -27,6 +27,7 @@
     "@cypress/request-promise": "^5.0.0",
     "@types/parse5-html-rewriting-stream": "5.1.1",
     "fs-extra": "9.1.0",
+    "mocha": "3.5.3",
```

Comment:
this package didn't have an explicit version defined in its devDependencies. I'm inferring that it was using the base package.json version and added it here explicitly.